GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -47,9 +61,36 @@
 // Expected path in the URL scheme to be handled.
 static NSString *const kBrowserCallbackPath = @"/oauth2callback";
 
+// The error code for Google Identity.
+NSErrorDomain const kGIDVerifyErrorDomain = @"com.google.GIDVerify";
+
+// Error string for user cancelations.
+static NSString *const kUserCanceledError = @"The user canceled the verification flow.";
+
+// Parameters in the callback URL coming back from browser.
+static NSString *const kOAuth2ErrorKeyName = @"error";
+static NSString *const kOAuth2AccessDenied = @"access_denied";
+
+// Minimum time to expiration for a restored access token.
+static const NSTimeInterval kMinimumRestoredAccessTokenTimeToExpire = 600.0;
+
+// The callback queue used for authentication flow.
+@interface GIDVerifyAuthFlow : GIDCallbackQueue
```

Comment:
I think I would prefer that this lives in its own set of files. Also, remember to replace `//` with `///` if you want a doc comment.

Reply:
Done.